src/libraries/pkg/baseline/packageIndex.json

```diff
@@ -522,6 +522,32 @@
         "5.0.0.0": "5.0.0"
       }
     },
+    "Microsoft.Extensions.DependencyModel": {
+      "StableVersions": [
+        "1.0.0",
+        "1.0.3",
+        "1.1.0",
+        "1.1.1",
+        "1.1.2",
+        "1.1.9",
+        "2.0.0",
+        "2.0.3",
+        "2.0.4",
+        "2.1.0",
+        "3.0.0",
+        "3.0.1",
+        "3.0.2",
+        "3.0.3",
+        "3.1.0",
+        "3.1.1",
+        "3.1.2",
+        "3.1.3"
+      ],
```

Comment:
Not strictly required but could you add a `"BaselineVersion": "5.0.0",` after the StableVersions as well?

Reply:
I can do it, but the other Microsoft.Extensions entries don't have it. What does it do?

Reply:
It will make sure that if any other package has a package dependency to this one, the dependency will be on the 5.0.0 package and not a previous version. We usually re-baseline every major release, we already did it for 5.0 but the extensions packages where added after that